shared/js/background/safari-events.es6.js

```diff
@@ -97,10 +93,42 @@ let handleMessage = (e) => {
     }
 }
 
+let getActiveTab = () => {
+    let activeTab = safari.application.activeBrowserWindow.activeTab
+    if (activeTab.ddgTabId) {
+        return tabManager.get({tabId: activeTab.ddgTabId})
+    } else {
+        let id = browserWrapper.getTabId({target: activeTab})
+        return tabManager.get({tabId: id})
+    }
+}
+
+let handleUIMessage = (req, res) => {
+    if (req.getCurrentTab || req.getTab) {
+        res(getActiveTab())
+    } else if (req.getTopBlocked) {
+        res(Companies.getTopBlocked(req.getTopBlocked))
+    } else if (req.getBrowser) {
+        res('safari')
+    } else if (req.whitelisted) {
+        res(tabManager.whitelistDomain(req.whitelisted))
+    } else if (req.getSiteScore) {
+        let tab = tabManager.get({tabId: req.getSiteScore})
+        if (tab) res(tab.site.score.get())
+    } else if (req.getTopBlockedByPages) {
+        res(Companies.getTopBlockedByPages(req.getTopBlockedByPages))
+    } else if (req.resetTrackersData) {
+        Companies.resetData()
+        safari.self.hide()
+    }
+}
+
+safari.extension.globalPage.contentWindow.message = handleUIMessage
+
 let updateSetting = (e) => {
     let name = e.message.updateSetting.name
     let val = e.message.updateSetting.value
-    if (name && val) {
+    if (name) {
```

Comment:
This change is not necessarily related to this PR, but I noticed that you can't turn off the twitter embed toggle in the options page, since `val` is falsy. I should've spotted this in review.

Reply:
Nice, thanks.